src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -526,6 +531,17 @@ internal ManagedCodePatterns(ManagedCodeConventions conventions)
                         {
                             new PatternDefinition("tools/{tfm}/{rid}/{any?}", table: AnyTable),
                     });
+
+                EmbedAssemblies = new PatternSet(
+                    conventions.Properties,
+                    groupPatterns: new PatternDefinition[]
+                        {
+                            new PatternDefinition("embed/{tfm}/{any?}", table: DotnetAnyTable),
+                        },
+                    pathPatterns: new PatternDefinition[]
+                        {
+                            new PatternDefinition("embed/{tfm}/{assembly}", table: DotnetAnyTable),
```

Comment:
We really should revisit the meaning of 'any' in our pattern definitions :) 

It's an inconsistent behavior right now, this is a note to myself, more than something relevant for this PR. :)